sdk/iot/azure-iot-modelsrepository/README.md

```diff
@@ -0,0 +1,16 @@
+# Azure IoT Models Repository Library
+
+The Azure IoT Models Repository Library for Python provides functionality for working with the Azure IoT Models Repository
+
+## Installation
+
+This package is not yet available on pip. Please install locally from source:
+
+```Shell
+python -m pip install -e <path to azure-iot-modelsrepository>
+```
+
+## Features
+
+* ### ModelsRepositoryClient
+    * Allows retrieval of model DTDLs from remote URLs or local filesystems
```

Comment:
I think "Allows the retrieval of model definitions..." or "Allows the retrieval of digital twin model definitions..." would be more clear.